mycroft/audio/main.py

```diff
@@ -124,15 +122,13 @@ def load_services(config, ws, path=None):
                 s = service_module.autodetect(config, ws)
                 service += s
             except Exception:
-                LOG.error('Failed to autodetect...',
-                          exc_info=True)
+                LOG.error('Failed to autodetect...')
```

Comment:
Should we at least print `e.__name__ + ': ' + str(e)` or something, here and below?

Reply:
Probably a good idea, yes